tests/schemas/invalid/test_invalid_placeholder_option_label_from_value.json

```diff
@@ -0,0 +1,97 @@
+{
+  "mime_type": "application/json/ons/eq",
+  "language": "en",
+  "schema_version": "0.0.1",
+  "data_version": "0.0.3",
+  "survey_id": "0",
+  "title": "Test option label from value",
+  "theme": "default",
+  "description": "A questionnaire to demo an invalid answer type for option_label_from_value transform",
+  "metadata": [
+    {
+      "name": "user_id",
+      "type": "string"
+    },
+    {
+      "name": "period_id",
+      "type": "string"
+    },
+    {
+      "name": "ru_name",
+      "type": "string"
+    }
+  ],
+  "questionnaire_flow": {
+    "type": "Linear",
+    "options": {
+      "summary": {
+        "collapsible": false
+      }
+    }
+  },
+  "sections": [
+    {
+      "id": "checkbox-section",
+      "groups": [
+        {
+          "blocks": [
+            {
+              "type": "Question",
+              "id": "name-question",
+              "question": {
+                "id": "primary-name-question",
+                "title": "Please enter body part where you had most serious injury",
+                "type": "General",
+                "answers": [
+                  {
+                    "id": "body-part-name",
+                    "label": "Body Part",
+                    "mandatory": true,
+                    "type": "TextField"
+                  }
+                ]
+              }
+            },
+            {
+              "type": "Interstitial",
```

Comment:
This schema is now invalid. It should only be invalid for the part that we want to be invalid, however, the JSON itself won't pass the validator as `Interstitial` cannot have `question`. To use interstitial, you need to use content block instead of question and pipe the placeholder into the content block's title etc.